src/bit-systems/object-spawner.ts

```diff
@@ -34,14 +32,6 @@ function* spawnObjectJob(world: HubsWorld, spawner: EntityID) {
   spawnerObj.updateMatrices();
   const spawnedObj = world.eid2obj.get(spawned)!;
   setMatrixWorld(spawnedObj, spawnerObj.matrixWorld);
-
-  yield* waitForMediaLoaded(world, spawned);
-  spawnerObj.visible = false;
-  yield sleep(1000);
-  spawnerObj.visible = true;
-
-  // TODO we should come up with a nicer way to get at the media that was loaded by a MediaLoader
-  yield* animateScale(world, spawnerObj.children[0]!.eid!);
```

Comment:
Question for my study: Here can be removed because of `animateLoad: true`?

Reply:
Yes, as we are already animating the spawned object I don't see the point on animating explicitly.